src/build/optimize-streams.ts

```diff
@@ -60,7 +60,7 @@ export class GenericOptimizeStream extends Transform {
       } catch (error) {
         logger.warn(
             `Unable to optimize ${this.validExtension} file ${file.path}`,
-            {err: error});
+            {err: error.message || error});
```

Comment:
If the problem is likely to be internal, `error.stack || error.message || error` is even better.

If the error message is most likely telling the user about a mistake that they made, then this form is better.